filters/include/pcl/filters/frustum_culling.h

```diff
@@ -192,6 +196,41 @@ namespace pcl
       {
         return (fp_dist_);
       }
+      
+      /** \brief Set the region of interest (ROI) in normalized values
+        *  
+        * This is an optional feature. If ROI is not set, 
+        * all points in the FOV of the camera are returned.
+        * Can be used to cut out objects based on 2D bounding boxes by object detection.
+        * 
+        * \param[in] roi_x X center position of ROI
+        * \param[in] roi_y Y center position of ROI
+        * \param[in] roi_w Width of ROI
+        * \param[in] roi_h Height of ROI
+        */
+      void 
+      setRegionOfInterest (float roi_x, float roi_y, float roi_w, float roi_h)
+      {
+        roi_x_ = roi_x;
```

Comment:
Would it make sense to check that the values doesn't go beyond a normalized value here? and maybe throw OutOfBounds exception or something like that?

Reply:
Then we can remove the checks in the hot loop?

Reply:
Moved ROI check to set function